docs/features/distribution.md

```diff
@@ -38,7 +38,26 @@ This module is in charge of distributing collected transaction fee and inflated
         
 3. Query reward token
 
-    Execute the command to get the earned tokens:
+    Execute the command to get the estimated inflation rewards :
+    
+    ```bash
+    iriscli distribution withdraw-rewards --from=bob  --dry-run --chain-id=test-irishub --fee=0.004iris
+    ```
```

Comment:
append `--commit`. Now by default, the broadcast type is `sync`

Reply:
fixed